app/Services/MovieApi.php

```diff
@@ -6,27 +6,36 @@
 
 class MovieApi
 {
-    https://api.themoviedb.org/3/discover/movie?api_key=f312ac2cb63002f508d52fd432cea28d&primary_release_date.gte=2017-09-25&language=en-US
-    const BASE_URL = 'https://api.themoviedb.org/3/discover/movie?';
+    
+    protected $base_url = 'https://api.themoviedb.org/3/discover/movie?';
     protected $api_key = 'f312ac2cb63002f508d52fd432cea28d';
     protected $language_key = 'en-US';
 
-    public function buildDate (){
-        
+    public function buildDate ()
+    {
+        $tz = 'America/New_York';
+        $timestamp = time();
+        $dt = new \DateTime("now", new \DateTimeZone($tz));
+        $dt->setTimestamp($timestamp);
+        return $dt->format('Y-m-d');
     }
 
-    public function buildURL($base, $key, $language, $time)
+    public function buildURL()
     {
-        $url = BASE_URL . 'api_key=' . $key . '&language=' . $language . '&primary_release_date.gte=' .;
+        $time = $this->buildDate();
+        $url = $this->base_url . 'api_key=' . $this->api_key . '&language=' . $this->language_key . '&primary_release_date.gte=' . $time;
```

Comment:
This works just for this scope of getting movies, but if you were to make calls to the api for a different url this method won't be modular enough to do that. Angel showed me how to use an array of parameters passed into the function, which makes the code a lot more accessible to other api url calls. If you want to check that out go to the buildApiUrl() method in my repo!